script/protocgen.sh

```diff
@@ -12,6 +12,14 @@ for dir in $proto_dirs; do
   --gocosmos_out=plugins=interfacetype+grpc,\
 Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
   $(find "${dir}" -maxdepth 1 -name '*.proto')
+
+  # command to generate gRPC gateway (*.pb.gw.go in respective modules) files
+  .cache/bin/protoc \
```

Comment:
Why are we using it from `.cache`?
```suggestion
  protoc \
```

Reply:
we're installing downloaded binaries into `.cache/bin` now.